src/Microsoft.AspNet.Hosting.Abstractions/EnvironmentName.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Hosting
+{
+    public static class EnvironmentName
+    {
+        public const string Development = "Development";
+        public const string Production = "Production";
```

Comment:
Great question. Part of me says to make it `static readonly` to avoid issues if we change these constants while we're in beta... though I admit that's a pretty weak reason. One unique benefit of having a `const` is that you can use it in attributes. Though we don't have any such attributes ourselves, it's conceivable that someone else might have such an attribute, e.g. `[StartupSelector(EnvironmentName.Production)]` or something.

But, having said all that, using `static readonly` is certainly "safer." Our general rule really is that `const` should be used for things that actually *can't* change, such as mathematical truths, or other intrinsic concepts (e.g. `const int MillisecondsPerSecond = 1000`).